monarch_mapping_commons.Makefile

```diff
@@ -40,3 +40,13 @@ $(MAPPING_DIR)/umls_hp.sssom.tsv:
 benchmark:
 	pip install py-spy
 	sudo py-spy record -o flamegraph.svg -- $(SSSOM_TOOLKIT) validate $(MAPPING_DIR)/gene_mappings.sssom.tsv
+
+
+$(TMP_DIR)/upheno/%:
+	mkdir -p $(TMP_DIR)/upheno/
+	wget https://bbop-ontologies.s3.amazonaws.com/upheno/current/upheno-release/all/$* -O $@
+
+$(MAPPING_DIR)/upheno_custom.sssom.tsv: $(patsubst %, $(TMP_DIR)/upheno/%, upheno_species_lexical.csv upheno_mapping_logical.csv upheno_all_with_relations.owl)
+	mkdir -p $(MAPPING_DIR) $(TMP_DIR)
+	phenio-toolkit lexical-mapping --species-lexical $(TMP_DIR)/upheno/upheno_species_lexical.csv -m $(TMP_DIR)/upheno/upheno_mapping_logical.csv -o $(TMP_DIR)
```

Comment:
It would be good to add the poetry run variable `$(RUN)` here too. I'm adding it to my PR.